HelpSource/themes/classic.css

```diff
@@ -0,0 +1,36 @@
+:root {
+    /* editor colors */
+    --cm-border-color: #e1e1e1;
+    --cm-background-color: #fff;
+    --cm-selection-color: #308cc6;
+    --cm-keyword-color: #0000e6;
+    --cm-built-in-color: #3333bf;
+    --cm-number-color: #980099;
+    --cm-symbol-color: #007300;
+    --cm-class-color: #0000d2;
+    --cm-primitive-color: #3333bf;
+    --cm-char-color: #007300;
+    --cm-env-var-color: #8c4614;
+    --cm-comment-color: #bf0000;
+    --cm-string-color: #5f5f5f;
+    --cm-text-color: #000;
+
+    /* derived colors
+    in future use `color-mix` to calculate them which allows for fewer variables
+    but currently the qt browser does not support it, see
+    https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/color-mix
+    */
+    --sc-bg-color: var(--cm-background-color);
+    --sc-bg-warning: var(--cm-selection-color);
+
+    --sc-text-color: var(--cm-text-color);
+    --sc-text-warning: var(--cm-keyword-color);
+    --sc-text-color-lighter: #333;
+
+    --sc-tertiary: var(--cm-built-in-color);
```

Comment:
Good idea! I agree. It makes sense to me that if one wants to change the "extension indicator", one could do it directly, without us needing to define a variable for it

Reply:
OK, I also think it is a good way. I deleted the variable.

Reply:
I think we got a bit confused on this one? Or at least I got confused:
didn't we decide to remove --color-extension-bg and --color-extension-fg, and put the default values directly in scdoc.css? like this
```css
.extension-indicator-ctr {
    background-color: #601c8b;
    color: white;
 }
```
Now I've opened an extension page in a non-default theme and I have seen a strange color, so I checked and we still have the variables and different values for every theme. I would say let's remove them all from themes, and put back default values directly in scdoc.css. Then as we said above, if someone wants to customize this they can write a simple rule for `.extension-indicator-ctr` in their `custom.css`.

Reply:
I deliberately put both colors as var now because I think it would be strange if every color is variable but the extension badge banner is static, so I decided to keep it variable and use some colors from the theme to customize it accordingly.

Let me know what you think about it or if we should really remove it.

Reply:
For me it would be ok either to ship the same "extension badge" style for all themes (aka delete the variables), or to touch up the current colors with this patch: https://gist.github.com/elgiano/c4e56a98a28889cd7b2140c97260c800

In the patch I use --color-link for extension background and white for text everywhere, to keep it in line with the original design, except in dracula.css, where white on light green is not readable, and I left your choice of --color-cm-char (with white text)

Reply:
Thx for your suggestions, I applied them in 0737d5e3f09f54bd3c5f580853402464031da77a